starters/next-wordpress-starter/components/page.js

```diff
@@ -0,0 +1,37 @@
+import Image from "next/image";
+import Link from "next/link";
+
+export default function Page({
+  page: { title, date, featuredImage, content },
+}) {
+  return (
+    <article className="prose lg:prose-xl mt-10 mx-auto">
+      <h1>{title}</h1>
+      <p className="text-sm text-gray-600">{new Date(date).toDateString()}</p>
+
+      <Link passHref href="/">
+        <a className="font-normal">Home &rarr;</a>
+      </Link>
```

Comment:
I think we want to "go back" here and link to `/pages` instead of home.

```suggestion
      <Link passHref href="/pages">
        <a className="font-normal">Pages &rarr;</a>
      </Link>
```